### PR TITLE
Certificates info screen update (EXPOSUREAPP-7661)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
Adapted certificates info screen (with new legal strings 🤩).

Screenshot 2021-06-15 at 15 32 38

### CODE REVIEW

**Comment by mtwalli** [comment]
the text in iOS is different, could you please check that ? and which one  is up to date ?

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml [hunk 1]
```diff
@@ -106,15 +106,17 @@
...
     <!-- XTXT: Vaccination Consent qr code text -->
     <string name="vaccination_consent_qr_info_qr_code_text">"Die Zertifikate gelten innerhalb der EU als gültiger Nachweis (z.B. für Reisen)."</string>
     <!-- XTXT: Vaccination Consent time text  -->
     <string name="vaccination_consent_qr_info_time_text">"Nach dem Hinzufügen bleiben die Zertifikate auf Ihrem Smartphone. Andere Personen können Ihre Daten nur einsehen, wenn Sie diesen ein Zertifikat zur Überprüfung vorzeigen."</string>
+    <!-- XTXT: Vaccination Consent family certificates text  -->
+    <string name="vaccination_consent_family_certificates_text">"Fügen Sie existierende Zertifikate für sich und Ihre Familienmitglieder hinzu."</string>
     <!-- XTXT: Text for vaccination consent legal information button -->
-    <string name="vaccination_consent_onboarding_legal_information">"Ausführliche Informationen zur Datenverarbeitung finden Sie in der Datenschutzerklärung."</string>
+    <string name="vaccination_consent_onboarding_legal_information">"Ausführliche Hinweise zur Datenverarbeitung finden Sie in der Datenschutzerklärung"</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml:119]: can you please move the strings to `covid_certificates_strings.xml` ? the plan to move all certificates related strings into that file

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by axelherbstreith — DISMISSED** [review]
lgtm - Darkmode Icons are also in place 👍

**Review by mtwalli — APPROVED** [review]
👍🏼  Thanks!

**Review by axelherbstreith — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**